composer.json

```diff
@@ -19,13 +19,15 @@
         "php": ">=5.3.2",
         "symfony/finder": ">=2.0",
         "symfony/yaml": ">=2.0",
-        "kriswallsmith/assetic": ">= 1.0",
-        "symfony/framework-bundle": ">=2.0.9"
+        "symfony/assetic-bundle": "^2.8",
+        "symfony/framework-bundle": "^2.3|^3.0",
+        "symfony/templating": "^3.3"
     },
     "require-dev": {
         "twig/extensions": "v1.0.0",
-        "squizlabs/php_codesniffer": "1.*",
-        "phpunit/phpunit": "3.7.*"
+        "squizlabs/php_codesniffer": "^2.7",
+        "phpunit/phpunit": "^5.7",
+        "phpunit/phpunit-mock-objects":"^3.4"
```

Comment:
This is already part of the `phpunit/phpunit` package, so there's no need to define it explicitly